src/script/components/LegalHoldDot.tsx

```diff
@@ -78,7 +78,7 @@ const LegalHoldDot: React.FC<LegalHoldDotProps> = ({
 
 export default LegalHoldDot;
 
-registerReactComponent('legal-hold-dot', {
+registerReactComponent<LegalHoldDotProps>('legal-hold-dot', {
```

Comment:
Not necessary but you can keep it if you like to.

```suggestion
registerReactComponent<LegalHoldDotProps>('legal-hold-dot', {
```